charts/cluster-autoscaler/templates/serviceaccount.yaml

```diff
@@ -3,11 +3,16 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   labels:
+    foo: bar
```

Comment:
I assume we don't want to ship the addition of a static `foo: bar` label?

Reply:
your assumption is correct 😅 , leftover while testing it.

Reply:
PR updated, good catch @jackfrancis !